std_msgs/String group
moveit_msgs/RobotTrajectory trajectory